Jenkinsfile_nightly

```diff
@@ -85,7 +85,6 @@ withNightlyPipeline(type, product, component) {
   before('fullFunctionalTest') {
     env.PARALLEL_CHUNKS = params.FUNCTIONAL_TESTS_WORKERS
     env.MOCKED_PAYMENTS = false
-    env.TEST_RETRIES = 2
```

Comment:
retries makes it much harder to investigate failures, as our tests are not independednt, and retries mess with data